llvm/lib/Transforms/Utils/SimplifyLibCalls.cpp

```diff
@@ -195,6 +195,12 @@ static void annotateNonNullAndDereferenceable(CallInst *CI, ArrayRef<unsigned> A
   }
 }
 
+static void mergeAttributes(CallInst *CI, AttributeList Attributes) {
```

Comment:
Submitted upstream as https://reviews.llvm.org/D135073 and https://reviews.llvm.org/D135075